src/test/ui/borrowck/borrowck-move-out-of-overloaded-auto-deref.stderr

```diff
@@ -9,9 +9,6 @@ LL |     let _x = Rc::new(vec![1, 2]).into_iter();
    |
 note: this function takes ownership of the receiver `self`, which moves value
```

Comment:
We should use `trim_short_path!(function_name)` instead of "function" in the note.